analyzer/throttle.c

```diff
@@ -61,15 +61,15 @@ suscan_throttle_get_portion(suscan_throttle_t *throttle, SUSCOUNT h)
   if (h > 0) {
     do {
       retry = SU_FALSE;
-      clock_gettime(CLOCK_MONOTONIC, &tn);
+      tn = suscan_gettime_raw();
 
-      timespecsub(&tn, &throttle->t0, &sub);
+      sub = tn - throttle->t0;
 
-      if (sub.tv_sec > 0) {
+      if (sub > 1000000000ull) {
```

Comment:
Declare a symbolic constant like ```SUSCAN_THROTTLE_LATE_READER_THRESHOLD_NS``` after the last include directive. Important to keep the ```_NS``` suffix.

Reply:
Ok.

Looking at it again, this condition changed by 1ns.  To get the same behavior, I should have used >=.  I like > a bit more as it read better "time is more than the threshold", so I'm going to leave it as >...unless you object.

Reply:
For thresholds in which the constant precision is below the uncertainty, it is not necessary to be that strict. I am okay with this.